Makefile-rpm-ostree.am

```diff
@@ -148,6 +148,10 @@ BUILT_SOURCES += $(binding_generated_sources)
 # Wraps `cargo test`.  This is always a debug non-release build;
 # the main thing here is we still drop the `target` dir in our build
 # directory, since we nominally support srcdir != builddir.
-rust-test:
-	cd $(top_srcdir)/rust && CARGO_TARGET_DIR=@abs_top_builddir@/target cargo test
-.PHONY: rust-test
+rust-test: $(binding_generated_sources) $(dbus_built_sources) $(librpmostree_rust_path) librpmostreecxxrs.la
+	cd $(top_srcdir)/rust && \
```

Comment:
I think the `cd $(top_srcdir)` should just be removed now.   And we also shouldn't need to set `CARGO_TARGET_DIR`.

Now, we probably do need the `LD_LIBRARY_PATH` wrapper, otherwise we will link against the installed `librpmostree-1.so`.

Reply:
(The `LD_LIBRARY_PATH` dance is something that libtool handles, but we no longer use libtool for the main binary)